### PR TITLE
imt/alejandra

### PR DESCRIPTION
**Pull request by alejandrasuperlano**
Tables and pagination components added

### CODE REVIEW

**Comment by cballevre** [comment]
@alejandrasuperlano thanks for the PR but as the code as already been merge with imt-team there no diff to watch. Keep it for the next modification you make

File: components/Table/StationTable.jsx [hunk 1]
```diff
@@ -1,10 +1,26 @@
 'use client'
 
 import { Table } from '@radix-ui/themes'
+import { useState } from 'react'
 import Link from 'next/link'
+import ReactPaginate from 'react-paginate';
 import { StationTableDropdown } from '../ui/Dropdown/StationTableDropdown'
+import '../ui/SearchBar/pagination-style.css';
```
Comment (felp99) [inline components/Table/StationTable.jsx:8]: This should being done by the tailwind in the classes of the components.

File: components/Table/StationTable.jsx [hunk 2]
```diff
@@ -38,6 +54,17 @@ export default function StationTable({ data }) {
           ))}
         </Table.Body>
       </Table.Root>
+      <ReactPaginate
+          previousLabel={'Previous'}
```
Comment (felp99) [inline components/Table/StationTable.jsx:58]: Use the translation function to translate the words for english and french

**Review by felp99 — APPROVED** [review]
In general is really good, just some point to correct in the next weeks.

**Review by felp99 — APPROVED** [review]

**Pull request merged**